Foundation/NSURLSession/http/EasyHandle.swift

```diff
@@ -660,3 +639,38 @@ internal extension CFURLSessionEasyCode {
         throw self
     }
 }
+
+#if os(Android)
+extension URLSession {
+    /// See https://curl.haxx.se/docs/sslcerts.html
+    /// For SSL to work you need a "cacert.pem" to be accessible at the
+    /// by setting URLSession.sslCertificateAuthorityFile to a file URL.
```

Comment:
This sentence is ungrammatical--"to be accessible at the <what???> by setting"...